eula.md

```diff
@@ -0,0 +1,43 @@
+# End-User License Agreement (EULA) of H.U.R.B.A.N. selector
```

Comment:
Could you please reflow the text in this file so that the longest line is shorter than 100 chars?